### ISSUE TITLE
Forms cannot be read by screen readers used by the visually impaired.

### ISSUE THREAD

**Issue by SpeakACCESS** [formtools/core#546]
The website for[ ACCESS Blind Tech](https://www.accessblindtech.com) was designed specifically for people who are blind or visually impaired so that it can easily be read with screen readers. When 'reading' the website using ACCESS, software developed to make it easier for the blind to use a computer, it worked beautifully except for the forms. 
The problem is that the screen reader cannot read the name of the fields, which means that a blind person has no way of knowing what information to enter. One possible solution is to prepopulate each field with the name of the field coded so that text entered automatically overwrites the field names.
